zebra-state/src/lib.rs

```diff
@@ -39,7 +39,9 @@ impl Config {
 impl Default for Config {
     fn default() -> Self {
         Self {
-            path: PathBuf::from("./.zebra-state"),
+            path: dirs::cache_dir()
+                .map(|dir| dir.join(".zebra").join("state"))
+                .unwrap_or_else(|| PathBuf::from(".zebra-state/")),
```

Comment:
Suggestion: The cache dir is already hidden, so we don't need to use `.zebra/state`. And we don't need a trailing slash.

```suggestion
                .map(|dir| dir.join("zebra").join("state"))
                .unwrap_or_else(|| PathBuf::from(".zebra-state")),
```